pandas/core/frame.py

```diff
@@ -4392,8 +4396,8 @@ def _dict_round(df, decimals):
                     raise ValueError("Index of decimals must be unique")
             new_cols = [col for col in _dict_round(self, decimals)]
         elif com.is_integer(decimals):
-            # Dispatch to numpy.round
-            new_cols = [np.round(v, decimals) for _, v in self.iteritems()]
+            # Dispatch to Series.round
+            new_cols = [v.round(decimals) for _, v in self.iteritems()]
```

Comment:
Actually, this change was needed to get rid of the ``out`` argument. If ``DataFrame.round()`` calls ``np.round()``, then ``np.round()`` dispatches to ``Series.round()``, passing the ``out`` argument.